cdc/sink/dmlsink/mq/worker.go

```diff
@@ -306,7 +306,8 @@ func (w *worker) sendMessages(ctx context.Context) error {
 				return errors.Trace(err)
 			}
 			for _, message := range future.Messages {
-				if message.ClaimCheckFileName != "" {
+				// w.claimCheck in pulsar is nil
```

Comment:
We are not aim to support claim-check functionality for the pulsar.

Reply:
When using pulsar, because w.claimCheck calls the method without checking whether it is empty, this is wrong, so you need to add a check if it is nil